pkg/cloud/services/eks/identity_provider.go

```diff
@@ -0,0 +1,129 @@
+package eks
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/eks"
+	"github.com/pkg/errors"
+	"sigs.k8s.io/cluster-api-provider-aws/controlplane/eks/api/v1alpha4"
```

Comment:
Best to replace with `infrav1exp "sigs.k8s.io/cluster-api-provider-aws/exp/api/v1alpha4"` for consistency

Reply:
Just a nice to have